docs/module/bgp.md

```diff
@@ -291,38 +291,47 @@ The BGP transformation module builds a list of BGP neighbors for every node. Tha
 ```
 
 (bgp-ebgp-sessions)=
 **EBGP sessions**
+
 * Whenever multiple nodes connected to the same link use different AS numbers, you'll get a full mesh of EBGP sessions between them[^EB_D].
 * Global (**bgp.as**) and local (**bgp.local_as**) autonomous systems are considered when deciding to create a session between two adjacent nodes, allowing you to create EBGP sessions between nodes belonging to the same AS or IBGP sessions between nodes belonging to different AS.
 * Parallel EBGP sessions are established for all IP address families configured on the link[^BSESS]. See also [IPv6 support](#ipv6-support).
 
 [^EB_D]: Unless you disabled EBGP sessions with the **bgp.sessions** parameter ([more details](bgp-advanced-node))
 
-See the [Simple BGP](bgp_example/simple.md) and [EBGP Data Center Fabric](bgp_example/ebgp.md) examples for more details.
+For more details, see the [Simple BGP](bgp_example/simple.md) and [EBGP Data Center Fabric](bgp_example/ebgp.md) examples.
+
+(bgp-ebgp-lla)=
+**IPv6 LLA Sessions**
+
+*netlab* creates an IPv6 LLA EBGP session (interface EBGP session) when the **unnumbered** link- or interface attribute is set, or when **ipv6** interface address or link prefix is set to *True* (indicating LLA-only IPv6 interface).
+
+Use the **netlab show module --module bgp --feature ipv6_lla** command to display devices on which you can use this feature.
 
-### Notes on Unnumbered EBGP Sessions
+(bgp-ebgp-rfc8950)=
+**EBGP Sessions on Unnumbered IPv4 Links**
 
-Unnumbered EBGP sessions are supported on a few platforms. *netlab* creates an IPv6 LLA EBGP session when the **unnumbered** link- or interface attribute is set, or when **ipv6** interface address or link prefix is set to *True* (IPv6 LLA).
+*netlab* can use an IPv6 EBGP sessions to transport IPv4 address family with IPv6 next hops (RFC 8950) -- the functionality commonly used to implement *interface EBGP sessions*. *netlab* will enable IPv4 AF over IPv6 LLA EBGP session when the **unnumbered** link- or interface attribute is set, or when **ipv4** interface address or link prefix is set to *True*.
 
-*netlab* can use an IPv6 LLA EBGP session to transport IPv4 address family with IPv6 next hops (RFC 8950) -- the functionality commonly used to implement *unnumbered EBGP sessions*. *netlab* will enable IPv4 AF over IPv6 LLA EBGP session when the **unnumbered** link- or interface attribute is set, or when **ipv4** interface address or link prefix is set to *True*.
+Use the **netlab show module --module bgp --feature ipv6_lla** command to display devices on which you can use the IPv4 address family on IPv6 LLA EBGP sessions and the **netlab show module --module bgp --feature rfc8950** command to display devices on which you can use the IPv4 address family on regular IPv6 EBGP sessions.
```

Comment:
Being pedantic here, but the distinction between ```ipv6_lla``` and ```rfc8950``` is a bit odd because RFC8950 style ipv6 next hops are used in both cases

```rfc8950_lla``` and ```rfc8950_numbered``` might make more sense?